pkg/operator/deploy/staticresources/master/deployment.yaml.tmpl

```
@@ -79,4 +90,14 @@ spec:
         operator: Exists
       - effect: NoSchedule
         operator: Exists
-
+      {{ if .UsesWorkloadIdentity }}
+      volumes:
+        - name: bound-sa-token
+          projected:
+          defaultMode: 420
+          sources:
+          - serviceAccountToken:
+              audience: openshift
+              expirationSeconds: 3600
+              path: token
```

YAML indentation error? we want all of this under the `projected` block, yea? ref https://kubernetes.io/docs/concepts/storage/projected-volumes/#example-configuration-secret-downwardapi-configmap

```suggestion
            defaultMode: 420
            sources:
            - serviceAccountToken:
                audience: openshift
                expirationSeconds: 3600
                path: token
```

---

I'm actually confused as to how CI/E2E passed with this - maybe I am wrong...

---

Maybe the conversion to json noticed the `:` after `projectd` and Just Did The Right Thing :tm:?

Either way, I've adjusted the indentation, even if only to make it more immediately apparent what's going on :laughing: